### PR TITLE
#12. The solution to Integer to Roman problem.

### PR DESCRIPTION
**Pull request by daniilnaumovich**

### CODE REVIEW

File: ScnSoft-Education/LeetCodeTraining/IntegerToRoman/IntegerToRomanSolution.cs [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+namespace LeetCodeTraining.IntegerToRoman
+{
+    //https://leetcode.com/problems/integer-to-roman/
+    public static class IntegerToRomanSolution
+    {
+        public static string IntToRoman(int num)
+        {
+            StringBuilder result = new StringBuilder();
+
```
Comment (shomzascnsoft) [inline ScnSoft-Education/LeetCodeTraining/IntegerToRoman/IntegerToRomanSolution.cs]: Extra line.

File: ScnSoft-Education/LeetCodeTraining/IntegerToRoman/IntegerToRomanSolution.cs [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+                {10,    "X"},
+                {9,     "IX"},
+                {5,     "V"},
+                {4,     "IV"},
+                {1,     "I"}
+            };
+
+            foreach (var rep in intToSymbol)
+            {
+                while (true)
```
Comment (shomzascnsoft) [inline ScnSoft-Education/LeetCodeTraining/IntegerToRoman/IntegerToRomanSolution.cs]: It seems the cycle can be simplified to:
```
            while (num >= rep.Key)
            {
                num -= rep.Key;
                result.Append(rep.Value);
            }
```

File: ScnSoft-Education/LeetCodeTraining/LeetCodeTraining.csproj [hunk 3]
```diff
@@ -1,4 +1,4 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
```
Comment (shomzascnsoft) [inline ScnSoft-Education/LeetCodeTraining/LeetCodeTraining.csproj:1]: It seems this changes are redundant, please pay attention what is included to a commit.

File: ScnSoft-Education/LeetCodeTraining/IntegerToRoman/IntegerToRomanSolution.cs [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+                {50,    "L"},
+                {40,    "XL"},
+                {10,    "X"},
+                {9,     "IX"},
+                {5,     "V"},
+                {4,     "IV"},
+                {1,     "I"}
+            };
+
+            foreach (var rep in intToSymbol)
```
Comment (shomzascnsoft) [inline ScnSoft-Education/LeetCodeTraining/IntegerToRoman/IntegerToRomanSolution.cs:28]: Can you please clarify if rep is a contraction for replacement?

Comment (daniilnaumovich) [reply ScnSoft-Education/LeetCodeTraining/IntegerToRoman/IntegerToRomanSolution.cs:28]: Hello @shomzascnsoft 
Actually it was intended as a contraction for "representation", but I believe "replacement" works there as well.

**Review by shomzascnsoft — APPROVED** [review]

**Pull request merged**